LoopKit/UserAlert.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  UserAlert.swift
+//  LoopKit
+//
+//  Created by Rick Pasetto on 4/8/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+/// Protocol that describes any class that handles Alert posting.
+public protocol UserAlertHandler: class {
+    /// Schedule the given alert for posting.
+    func scheduleAlert(_ alert: UserAlert)
+    /// Unschedule any pending alerts with the given identifier.
+    func unscheduleAlert(identifier: UserAlert.Identifier)
```

Comment:
unscheduleAlert and cancelAlert -> removePendingAlertRequest and removeDeliveredAlert?

Reply:
I changed it to `removePendingAlert` and `removeDeliveredAlert`...hope that's ok, it felt more symmetric.